Makefile

```diff
@@ -111,4 +111,4 @@ rundev:
 	cd scripts && ./run.sh
 
 run:
-	docker-compose -f docker/docker-compose.yml up
+	docker-compose -f docker/docker-compose.yml up -d
```

Comment:
Why are we running as a daemon now?

Reply:
Because there is no point in blocking the terminal as there are too many services and too many logs to track them from the single output.